index.html

```diff
@@ -160,9 +158,9 @@ <h2>
       </p>
       <aside class="note" title="Common issuer identification numbers (IIN)">
         <p>
-          Issuer identification numbers (IIN) are maintained by ISO but are not
-          available to the general public. However, Wikipedia maintains a list
-          of the common <a href=
+          <dfn data-abbr="">Issuer identification numbers</dfn> are maintained
```

Comment:
Hi @marcoscaceres, any reason you dropped "(IIN): ?

Reply:
Ah, it looks like you might have planned to with data-abbr but the value is nil...whereas below the value is non-nil (for PCI)....

Reply:
Gets automatically added :)